lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1546,7 +1546,7 @@ func (w *WalletKit) fundPsbtCoinSelect(account string, changeIndex int32,
 
 		changeAmt, needMore, err := chanfunding.CalculateChangeAmount(
 			inputSum, outputSum, packetFeeNoChange,
-			packetFeeWithChange, changeDustLimit, changeType,
+			packetFeeWithChange, changeDustLimit, changeType, true,
```

Comment:
I think adding this flag makes sense, as you might want to be able to craft a 1-in-1-out transaction that pays more than 20% fees when it's a small input amount. But I think we should also expose this to the RPC and not just set this to `true` statically here.
So a new bool RPC parameter, for example `allow_high_fees` which by default is `false`.

Reply:
Ok, very happy to see that you agree with this approach.

I didn't expose this on RPC only because of "bloating" concerns, as I didn't want this extra flag to reach the user-facing params.
But I also think a psbt related call is the place to be specific and have fine control, on the other hand. So will change it as per your suggestion :ok_hand: